eng/common/docgeneration/Generate-DocIndex.ps1

```diff
@@ -188,5 +188,6 @@ if ($GetGithubIoDocIndexFn -and (Test-Path "function:$GetGithubIoDocIndexFn"))
 }
 else
 {
-    LogWarning "The function 'GetGithubIoDocIndexFn' was not found."
+    LogWarning "The function 'GetGithubIoDocIndexFn' was not found.`
+    Make sure it is present in eng/common/scripts/common.ps1 and eng/scripts/Language-Settings.ps1"
```

Comment:
It doesn't need to be in common.ps1. These functions should only need to be present in Language-Settings.ps1.